.github/ISSUE_TEMPLATE/design-intent-collaboration.md

```diff
@@ -7,51 +7,90 @@ assignees: shiragoodman, allison0034
-- [ ] VFS: Respond appropriately to Platform feedback tickets.
-- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061).
-- [ ] VFS: Close this ticket when all steps above are completed.
+# Steps to complete Design Intent Collaboration
+
+## Before meeting
+
+**VFS Product Manager or Designer**
+- [ ]  Create this issue and notify the Collaboration Cycle team of this ticket in the in #vfs-platform-support ([see Slack Workflow (User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)). 
+- [ ] Add artifacts listed below to the ticket at least 2 days before the scheduled meeting.
+
+**Platform**
+- [ ] Schedule meeting with Platform reviewers and VFS team attendees listed in ticket. 
+
+## After meeting
+
+**Platform**
+- [ ] Create feedback tickets and link to the Design Intent epic (if no feedback, please comment stating that). 
+- [ ] Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061).
+
+**VFS**
+- [ ] Respond to Platform feedback tickets.
+- [ ] Close this ticket when review is complete and feedback has been addressed. 
+
 
 ### Has your product been reviewed by the OCTO-DE Design Lead?
 - [ ] Yes
 - [ ] No
 
 
 ## Artifacts
-- Link to [product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/product-outline-template.md)
-- Whiteboard sketch or other lo-fi prototypes or wireframes
-- User flow
-- Any other artifacts you have so far
-
-## Meeting attendees from requesting team
-- Designer(s) (required): name(s)
-- Product Manager (required): name
-- OCTO-DE Design Lead (required): Ryan Thurlwell
-- OCTO-DE Product Lead (recommended): name
-- VFS content, IA, and QA specialists (if applicable): name(s)
-- Other stakeholders you want to invite (if applicable): name(s)
+
+Provide links or documents for the following: 
+
+- [ ] Product outline
+ 
+<!--
+[Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/product-outline-template.md)
+--> 
+ 
+- [ ] Whiteboard sketch or other lo-fi prototypes or wireframes
+- [ ] User flow
+- [ ] Any other artifacts you have so far
+
+
+## VFS Meeting Attendees
+
+### Required
+- Product Manager: [Insert name here]
+- OCTO-DE Product lead: [Insert name here]
+- OCTO-DE Design lead: Ryan Thurlwell
+
+### Team Members
+<!-- Add anyone from your team who significantly contributed to the artifacts, otherwise, delete list -->
+- [Insert name here]
+- [Insert name here]
+
+### VFS Attendees
+<!-- Delete any of the below if they are not applicable for your meeting -->
+- Content and IA: 
+- Accessibility: 
+- QA specialists: 
+
 
 ## Scheduling
 
-### Design Intents can only be scheduled on Monday, Tuesday, or Friday. 
```

Comment:
did we remove "Design intents can only be scheduled on monday , tuesday and Friday." I would like to see this included as a comment.

Reply:
Hi, @shiragoodman it's still in there.. it's part of the content note that shows conditionally with the other scheduling bullet points. 

<!-- 
Please note:
- Design Intents are scheduled on Monday, Tuesday, or Friday.
- All times are shown in ET. 
- Available time slots are indicated with [REVIEW BLOCK]. 
- When you reserve a time slot, the reserved meeting will show up alongside the [REVIEW BLOCK]. 
- Please select a time slot at least 2 business days from request date.
-->